Life of an Ad Request.md

```diff
@@ -103,40 +102,120 @@ The seller front end will send one request to the TEEs of each DSP included in t
 - Buyer Front End Service: receives the request, fetches real time bidding signals, performs the merge functions, and invokes the generateBid functions by sending a request to a Bidder Service.
 - Bidding Service: handles the execution of the bidding process and returning 0 or more bids to the buyer front end (BFE).
 
-## Flow
-1. The BFE receives the request and merges the owners IGs as indicated. It does this by invoking the `mergeFunction` defined in the IGs `biddingFunctionUrl` (or the default merge function if none is defined), passing in any IG for that owner with the same `biddingFunctionUrl`. The `mergeFunction` can return 0 or more IGs:
-    - If no IGs are returned, each IG joined in the browser will participate in the auction on its own.
-    - If 1 or more IGs are returned, only those IGs will participate in the auction.
 
-2. The BFE makes a call to the DSPs key-value (KV) server as indicated by the `trustedBiddingSignalsUrl` and `trustedBiddingSignalsKeys` from our remaining IG definitions. Examples of Real Time Data the DSP might want to fetch include:
-    - Budget and pacing data, which must stay up to date per campaign across all browsers.
-    - Campaign settings, such as an active status, updated targeting, etc.
-    - Creatives to render and have K-checked.
+## Biddding WorkFlow
+Schema below describe end to end workflow for private auction.
+<span style="display:block;text-align:center">![Ad Selection workflow](images/adsapi_end_to_end_workflow.png)</span>
+ 
+
+- **1.1.** SellerFrontEnd sends request to BuyerFrontEnd. 
+<br>**BFE - KVS**
+- **2.1.** BuyerFrontEnd calls Key-Value-Service.
+- **2.2.** Key-Value-Service sends response to BFE. 
+<br>**BFE – Bidding Service**
+- **3.1.** BFE sends request to Bidding Service. 
+- **3.2.** Bidding Service creates [Invocation Requests](https://github.com/privacysandbox/data-plane-shared-libraries/blob/a42a0702c7af9b001e0609f604988ed21541abbd/src/roma/interface/roma.h#L75) for the Roma Service to execute `generateBid` JavaScript function. Interest Groups are part of Invocation Request and by default they are combined it into one request. After that request is sent to Roma Service.

```

Comment:
yes you right. 
Anyway, it's important to understand how it works. For example, it will be helpful when someone will try to understand why this steps takes latency and cpu.